packages/@react-native-windows/cli/src/runWindows/utils/autolink.ts

```diff
@@ -401,6 +411,48 @@ async function updateAutoLink(
         changesNecessary;
     }
 
+    // Generating props for app project consumption
```

Comment:
Would be great if we could add some unit tests for this, not that the CLI lets you add them pretty easily. This logic has gotten pretty tricky, so I think they could help a lot.

Reply:
+1 
We also have issue #5882 tracking creating a sample native module and adding an automated test to ensure autolinking works.  I'd love to see that happen as it will help guarantee autolinking works end-to-end.

---
In reply to: [514454496](https://github.com/microsoft/react-native-windows/pull/6375#discussion_r514454496) [](ancestors = 514454496)

Reply:
Added unit tests

Reply:
Thank you for adding these!